cmd/proxy/cmd/proxy.go

```diff
@@ -89,6 +89,9 @@ type ClusterChecker struct {
 	endPollonProxyCh chan error
 
 	pollonMutex sync.Mutex
+
+	checkIntervalSeconds  int
+	requestTimeoutSeconds int
```

Comment:
Please call them as the config options: ProxyCheckInterval and ProxyTimeoutInterval.